src/background/stores/assessment-store.ts

```diff
@@ -442,7 +442,10 @@ export class AssessmentStore extends PersistentStore<AssessmentStoreData> {
             this.generateDefaultState(null),
         );
         this.state.assessments[test.key] = defaultTestStatus;
-        this.state.assessmentNavState.selectedTestSubview = test.requirements[0].key;
+
+        if (this.state.assessmentNavState.selectedTestType === payload.test) {
```

Comment:
Hm, reading through the description, I'm not sure I follow this. When would this not be true?

Reply:
I was running into a bug where the startOver message would be triggered from the issues table right before the details view switched to a different assessment requirement. So it would cause us to get into a state where the selected test type was something like `landmarks` or some other assessment requirement and the selected test subview was a rule not associated with that requirement. This is a fix for that bug.